decidim-core/app/cells/decidim/collapsible_list/show.erb

```diff
@@ -15,14 +15,14 @@
       <span class="collapsible-list__see-less" data-toggle="collapsible-list-<%= seed %>"><%= t("decidim.collapsible_list.see_less") %></span>
     </span>
   </div>
-<% else %>
-  <div class="collapsible-list <%= list_class %>">
-    <% list.each do |element| %>
-      <% if cell_name %>
-        <%= cell cell_name, element, cell_options %>
-      <% else %>
-        <%= card_for element, cell_options %>
+  <% else %>
```

Comment:
We usually try to respect the indentation as it was before the change, so it's easier to work with git tools (like `git diff` to see changes between commits/branches, or `git blame` to see who made a change in a file line-by-line). 

Can you leave out the indentation as it was before, so we only have the relevant change (line 19) on this PR?